frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.tsx

```diff
@@ -10,6 +13,7 @@ import {
   DashboardItemProps,
   withDashboardResources,
 } from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { DashboardsStorageCapacityDropdownItem } from '@console/plugin-sdk';
```

Comment:
Note that this changes with #2592.

Plugin-specific extension types should live inside the corresponding plugin pkg.